haproxy/README.md

```diff
@@ -69,3 +69,106 @@ This package does not currently support high availability on its own.
 ## Monitoring
 
 This service can be monitored by making TCP connections to the configured port (default is 80)
+
+# Testing haproxy
+
+Since the haproxy does not run in isolation but must be bound to a backend webserver, the following instructions exercise it in a simple scenario, one haproxy and one backend web server.
+
+### Build the haproxy core plan
+
+At a terminal build the core/haproxy
+
+```bash
+hab pkg build .
+
+# copy the last_build.env because it gets overwritten
+# in later steps and is required for testing later
+cp results/last_build.env results/last_build_haproxy.env
```

Comment:
Suggest having the instructions be to `cd` into the test plan directories and build there so each plan has a separate `results` directory; instead of multi-purposing one results directory.